app/assets/javascripts/components/dropdown-menu.js

```diff
@@ -0,0 +1,31 @@
+ManageIQ.angular.app.component('dropdownMenu', {
+  bindings: {
+    id: '<',
+    buttons: '<',
+  },
+  controllerAs: 'vm',
+  controller: function() {
+    var vm = this;
+
+    this.$onInit = function() {
+      vm.dropdown_id = 'btn_' + vm.id;
+      vm.buttons = JSON.parse(vm.buttons);
```

Comment:
missing $onChange probably .. or should the binding be `@` since we're passing in a string?